clientSide/src/components/EditCustomer/EditCustomer.jsx

```diff
@@ -0,0 +1,92 @@
+import { useNavigate, useParams, Link } from "react-router-dom";
+import NavBar from "../NavBar/Navbar";
+import GrayColumn from "../UiComponents/GrayColumn";
+import GrayContainer from "../UiComponents/GrayContainer";
+import LabeledInput from "../LabeledInput/LabeledInput";
+import { onChangeIntHandler } from "../../functions/onChangeHandlers";
+import { useState } from "react";
+import {
+  CustomerInfoLabels,
+  GeneralLabels,
+} from "../../text-labels/text-labels";
+import Button from "../Button/Button";
+import PageTitle from "../PageTitle/PageTitle";
+import { onChangeHandler } from "../../functions/onChangeHandlers";
+import "./EditCustomer.css";
+import { updateCustomer } from "../../parse-functions/updateFunctions";
+
+const EditCustomer = () => {
+  const { bookingId, customerId } = useParams();
+  const navigate = useNavigate();
+  const [formData, setFormData] = useState();
+
+  return (
+    <>
+      <NavBar />
+      <PageTitle ptitle="Edit customer"></PageTitle>
+      <form
+        onSubmit={(e) =>
+          updateCustomer(e, customerId, bookingId, formData, navigate)
+        }
+      >
+        <GrayContainer>
+          <GrayColumn>
+            <LabeledInput
+              labelText={CustomerInfoLabels.firstName}
```

Comment:
I think you use exactly the same fields with the Customer Info Component? 
Why not importing it instead of pasting the code it has?